pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -363,6 +375,26 @@ func updateNetwork(vmi *k6tv1.VirtualMachineInstance, vmStats *stats.DomainStats
 	}
 }
 
+func updateTelemetry(nodeName string, vmis []*k6tv1.VirtualMachineInstance, ch chan<- prometheus.Metric) {
+	phasesMap := make(map[string]uint64)
+
+	for _, vmi := range vmis {
+		phasesMap[strings.ToLower(string(vmi.Status.Phase))] += 1
+	}
+
+	for phase, count := range phasesMap {
```

Comment:
There seem to be no explicit unit tests to cover that summing them per status works.

Reply:
Same point @qinqon raised. I wanted to fix all the tests running multiple VMIs (https://github.com/kubevirt/kubevirt/pull/2512) but looks like I bited more than I can chew :) I'll quickly fix this test and keep working on #2512 to improve the testsuite later.

Reply:
I am more concerned about the missing unit tests here, not so much about the functional tests.

Reply:
ack, adding